doc/nestml_language/nestml_language_concepts.rst

```diff
@@ -759,13 +765,12 @@ For any two valid numeric expressions ``a``, ``b``, boolean expressions ``c``,\
 Blocks
 ------
 
-To structure NESTML files, all content is structured in blocks. Blocks begin with a keyword specifying the type of the block followed by a colon. They are closed with the keyword ``end``. Indentation inside a block is not mandatory but recommended for better readability. Each of the following blocks must only occur at most once on each level. Some of the blocks are required to occur in every neuron model. The general syntax looks like this:
+To structure NESTML files, all content is structured in blocks. Blocks begin with a keyword specifying the type of the block followed by a colon. Indentation inside a block is not mandatory but recommended for better readability. Each of the following blocks must only occur at most once on each level. Some of the blocks are required to occur in every neuron model. The general syntax looks like this:
```

Comment:
> Indentation inside a block is not mandatory but recommended for better readability.

I think this is wrong now, isn't it? I understood it such that the code inside of blocks **must** be indented now.